docs/tutorials/openvino-mlflow.md

```diff
@@ -0,0 +1,611 @@
+
+With the successful execution of the workflow, three new FuseML applications should have been created, one for each available prediction serving platform. Listing the FuseML applications will show the URLs where the prediction servers can receive inference requests:
+
+```bash
+fuseml application list
++----------------+-----------+---------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------+------------------+
+| NAME           | TYPE      | DESCRIPTION                                                                                             | URL                                                                                                                       | WORKFLOW         |
++----------------+-----------+---------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------+------------------+
+| cifar10-kserve | predictor | KServe service deployed for s3://mlflow-artifacts/1/7b1549fc8c3a45189564c521f9313a66/artifacts/model    | http://cifar10-kserve.fuseml-workloads.34.118.93.249.nip.io/v1/models/cifar10-kserve:predict                              | mlflow-multi-e2e |
+| cifar10-ovms   | predictor | OVMS service deployed for s3://mlflow-artifacts/1/7b1549fc8c3a45189564c521f9313a66/artifacts/model/ovms | http://cifar10-ovms.ovms.34.118.93.249.nip.io/v1/models/default:predict                                                   | mlflow-multi-e2e |
+| cifar10-seldon | predictor | Application generated by mlflow-multi-e2e workflow                                                      | http://cifar10-seldon.seldon.34.118.93.249.nip.io/seldon/fuseml-workloads/cifar10-seldon/v1/models/cifar10-seldon:predict | mlflow-multi-e2e |
++----------------+-----------+---------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------+------------------+
+```
+
+The list of FuseML applications include a URL to query the model for predictions. In addition to the URLs, since our workflow includes a KServe prediction service, you can also check the deployed models through the KServe
+dashboard (`http://kserve-web-app.<FUSEML_DOMAIN>/`, in our case `http://kserve-web-app.34.118.93.249.nip.io/`) which also includes more detailed information, such as the status of the deployment, logs, etc.
+
+For example:
+
+<figure markdown> 
+  ![KSserve Dashboard](img/openvino-mlflow/kserve.png)
+  <figcaption>KFServing Dashboard</figcaption>
+</figure>
```

Comment:
Some comment about those images, it is really hard to see anything without zooming in.

Reply:
fixed